queries.sql

```diff
@@ -24,3 +24,51 @@ SELECT * from animals WHERE NOT name='Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * from animals WHERE weight_kg>=10.4 AND weight_kg<=17.3;
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+ALTER TABLE animals ADD species VARCHAR(50);
```

Comment:
- [ ] Kindly place this statement in `data.sql` as this is a statement that alters the schema of the database